### ISSUE TITLE
Bad Block Reward

### ISSUE THREAD

**Issue by pool2mine** [sammy007/open-ethereum-pool#437]
Hi, 

I updated unlocker.go as deke997 suggested but still getting blocks marked as 3ETH. can i just change the value of byzantiumReward from **3**000000000000000000 to **2**000000000000000000 so i can get rid of this issue?

please let me know.

**Comment by mathias7799** [comment]
Yep and then you need to change it in www/config/environment.js for it to show correctly on the frontend.

**Comment by pool2mine** [comment]
and build again, right?

**Comment by pool2mine** [comment]
@mathias7799 can you tell me what should i modify in environment.js? because i don't see anything referring to block rewards in that file. thanks!

**Comment by njmdk** [comment]
change the value of byzantiumReward from 3000000000000000000 to 2000000000000000000
